### PR TITLE
Chore: (Docs - Sharing) Adds documentation on design integrations

### PR DESCRIPTION
**Pull request by jonniebigodes**
With this pull request, the documentation is updated to include information on third-party design tools integrators which support Storybook.

What was done:
- Sidebar toc was updated for the new documentation
- Created the relevant documentation with the necessary assets.

Feel free to provide feedback

### CODE REVIEW

File: docs/sharing/design-integrations.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+![Storybook design integrations](./storybook-design-integrations.jpeg)
+
+For development teams, Storybook provides the necessary tools to build, [test](../writing-tests/introduction.md), and [document](../writing-docs/introduction.md) UI components. [Integrating](https://storybook.js.org/addons/tag/design/) your Storybook with popular design tools (for example, [Figma](https://www.figma.com/), [Zeplin](https://zeplin.io/)) unlocks new workflows for you and your team. You prototype the UI in a design tool, build and test it in Storybook, allowing you to surface issues and ensure consistent and reliable UIs.
+
+## Figma
+
+[Figma](https://www.figma.com/) is a popular collaborative UI design tool used by [Slack](https://slack.com/), [Twitter](https://twitter.com/), and [Dropbox](https://www.dropbox.com/) teams. It allows multiple people to work on the same design simultaneously in the browser.
+
+If you're working with Figma, you have two options for connecting your Storybook. You can use the [Storybook addon designs](https://storybook.js.org/addons/storybook-addon-designs), which allows you to preview your designs alongside the currently selected story.
```
Comment (winkerVSbecks) [inline docs/sharing/design-integrations.md:15]:
```suggestion
If you're working with Figma, you have two options for connecting your Storybook. You can use the [Storybook addon designs](https://storybook.js.org/addons/storybook-addon-designs), which allows you to view your designs alongside the currently selected story.
```

File: docs/sharing/design-integrations.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+![Storybook addon figma](./storybook-figma-addon.png)
+
+Or via a plugin developed by the Storybook maintainers. [Storybook Connect](https://www.figma.com/community/plugin/1056265616080331589/Storybook-Connect) is the official Figma plugin that lets you link stories to your Figma components. You can preview your existing stories in the Figma design workspace when configured. Below is an overview of how to get started with the plugin.
```
Comment (winkerVSbecks) [inline docs/sharing/design-integrations.md:19]: Simplified the wording and cut out the bit about it being an official plugin. IMO if its going in the docs then its official by default. 

```suggestion
Or use [Storybook Connect](https://www.figma.com/community/plugin/1056265616080331589/Storybook-Connect) to embed stories into your Figma design workspace. Below is an overview of how to get started with the plugin.
```

File: docs/sharing/design-integrations.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+<video autoPlay muted playsInline loop>
+  <source src="figma-plugin-open-story.mp4" type="video/mp4" />
+</video>
+
+## Adobe XD
+
+[Adobe XD](https://www.adobe.com/products/xd.html) is a vector-based UI and UX design tool, enabling design teams to create wireframes, interactive designs, prototypes, or hi-fidelity web or application designs.
+
+Adobe XD integrates with Storybook via [addon](https://storybook.js.org/addons/storybook-addon-xd-designs), enabling you to embed your design specs alongside your Storybook stories.
```
Comment (winkerVSbecks) [inline docs/sharing/design-integrations.md:59]: This addon is no longer maintained. I was able to get it working using the iframe option of the designs addon: https://pocka.github.io/storybook-addon-designs/?path=/story/docs-iframe-readme--page

File: docs/sharing/design-integrations.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
+---
+title: 'Design integrations'
+---
+
+Storybook supports first-class integrations with well-known design tools, speeding up the development cycle by enabling you to cross-reference designs with your existing stories.
+
+![Storybook design integrations](./storybook-design-integrations.jpeg)
+
+For development teams, Storybook provides the necessary tools to build, [test](../writing-tests/introduction.md), and [document](../writing-docs/introduction.md) UI components. [Integrating](https://storybook.js.org/addons/tag/design/) your Storybook with popular design tools (for example, [Figma](https://www.figma.com/), [Zeplin](https://zeplin.io/)) unlocks new workflows for you and your team. You prototype the UI in a design tool, build and test it in Storybook, allowing you to surface issues and ensure consistent and reliable UIs.
```
Comment (kylegach) [inline docs/sharing/design-integrations.md:9]: Just a suggestion:

```suggestion
For development teams, Storybook provides the necessary tools to build, [test](../writing-tests/introduction.md), and [document](../writing-docs/introduction.md) UI components. Integrating your Storybook with popular design tools (for example, [Figma](https://www.figma.com/), [Zeplin](https://zeplin.io/), and [more](https://storybook.js.org/addons/tag/design/)) unlocks new workflows for you and your team. You prototype the UI in a design tool, build and test it in Storybook, allowing you to surface issues and ensure consistent and reliable UIs.
```

Comment (kylegach) [inline docs/sharing/design-integrations.md:9]: > You prototype the UI in a design tool, build and test it in Storybook, allowing you to surface issues and ensure consistent and reliable UIs.

There's a missing word here, I think.

File: docs/sharing/design-integrations.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+![Figma palette Storybook connect](./figma-plugin-open-in-figma.png)
+
+Follow the instructions to connect and authenticate with Chromatic.
+
+### Link stories to Figma components
+
+The core feature of the [Storybook Connect](https://www.figma.com/community/plugin/1056265616080331589/Storybook-Connect) Figma plugin is the ability to connect your existing components, variants, and instances to Storybook stories.
+
+To link your components with Storybook stories, navigate to the story published on Chromatic, ensure that you're using the correct branch, and copy the link.
```
Comment (kylegach) [inline docs/sharing/design-integrations.md:35]: The current constructions implies that these are linear steps, but they're not.

```suggestion
To link your components with Storybook stories, navigate to the story published on Chromatic (be sure that you're using the correct branch) and copy the link.
```

File: docs/sharing/design-integrations.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+In Figma, open the plugin and paste the URL.
```
Comment (kylegach) [inline docs/sharing/design-integrations.md:37]: > Open the plugin

These instructions skip [Step 1 of the Chromatic docs' instructions](https://www.chromatic.com/docs/figma#link-a-story-to-a-figma-component):

> Select a Figma component to link.

**Review by kylegach — APPROVED** [review]
Great work! It's really cool to see all of these in one place.

File: docs/sharing/design-integrations.md (hunk 1 above)
Comment (jonniebigodes) [reply docs/sharing/design-integrations.md]: Updated with the latest commit

File: docs/sharing/design-integrations.md (hunk 2 above)
Comment (jonniebigodes) [reply docs/sharing/design-integrations.md]: Update with the latest commit. Thanks for the suggestion @winkerVSbecks

File: docs/sharing/design-integrations.md (hunk 4 above)
Comment (jonniebigodes) [reply docs/sharing/design-integrations.md]: Thanks for the suggestion @kylegach I've factored it in with the latest commit.

File: docs/sharing/design-integrations.md (hunk 3 above)
Comment (jonniebigodes) [reply docs/sharing/design-integrations.md]: @winkerVSbecks thanks for the callout on this! Appreciate it :)! I've updated the section accordingly.

File: docs/sharing/design-integrations.md (hunk 6 above)
Comment (jonniebigodes) [reply docs/sharing/design-integrations.md]: Updated with the latest commit.

File: docs/sharing/design-integrations.md (hunk 4 above)
Comment (jonniebigodes) [reply docs/sharing/design-integrations.md]: Don't think so. I double-checked it and it panned out!

**Comment by jonniebigodes** [comment]
merging this!

**Pull request merged**